cohort_4/playbook/ACTIVITIES.md

```diff
@@ -88,6 +89,109 @@
 
 <!-- markdown-toc end -->
 
+# Creating a safe space
+
+The one rule that we have at Hack Camp is that we make it a safe space.
+We all know how bad it feels when we are made fun of or when someone makes
+us feel uncomfortable.
+
+Below are some scenarios which may occur and how to handle them:
```

Comment:
I'm saying that you wrote:

> Below are some scenarios which may occur and how to handle them:

But right now there's only once scenario listed. I think keeping it as is because we may add things to it later is a case of premature optimization and makes the writing less clear.